client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import HeaderCake from 'components/header-cake';
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+import FilePicker from 'components/file-picker';
+import DropZone from 'components/drop-zone';
+import { localize } from 'i18n-calypso';
+import notices from 'notices';
```

Comment:
Huh, interesting. Okay, turns out I don't really have a clue which is the preferred way these days. Maybe worth dropping a quick question in the Calypso Slack channel so we know from now on. Non-blocking tho :-)